### ISSUE TITLE
Vite stops working after plugin installation

### ISSUE THREAD

**Issue by denyncrawford** [stafyniaksacha/vite-plugin-fonts#1]
Well I don't know what to do so I ask here to find out if there is a reason this is happening.

The problem is that just by installing this plugin my app with vite does not start anymore, if I uninstall it, it keeps showing the message: Error: Cannot find module 'cac'. 

My solution was worse, which was to uninstall and reinstall all the modules, now it seems that it starts but does not work, and when doing build I get a rollup error, not even cloning the repo seems to work .-. :/

I don't think the second is something related to the plugin, I was probably using a version of vite compatible with other modules but now since it is not, I get the error.

The question is, what could have caused it?

**Comment by stafyniaksacha** [comment]
Hum, this seems to be a side effect with something else.
Can you tell me a bit more about your vite configuration ? (plugins used, details about your environment, which package manager you are using)

**Comment by denyncrawford** [comment]
Yep, this is my config file: 

```javascript
import { join } from 'path'
import { UserConfig } from 'vite'
import dotenv from 'dotenv'
import vue from '@vitejs/plugin-vue'

dotenv.config({ path: join(__dirname, '.env') })
const root = join(__dirname, 'src/render')

const config: UserConfig = {
  root,
  resolve: {
    alias: {
      '/@/': root,
    }
  },
  base: './',
  build: {
    outDir: join('../../dist/render'),
    emptyOutDir: true,
    assetsInlineLimit: 0
  },
  server: {
    port: +process.env.PORT,
  },
  plugins: [
    vue()
  ],
  optimizeDeps: {
    exclude: [
      'electron-is-dev',
      'electron-store',
    ]
  },
}

export default config
```

The problem of not being able to make the app work was because it had vite with pointed to the latest version in the package.json file and when I had to reinstall the modules it was updated to vite 2 and the configuration I had was no longer compatible

Anyway, something caused the failure and it was related to the installation of this plugin :/ I have not tried to install again, I ended up downloading the fonts and using them manually.

**Comment by stafyniaksacha** [comment]
I updated all dependencies, and published the `0.2.0` version.
Can you tell me if you still have the issue ?

**Comment by stafyniaksacha** [comment]
I close the issue since inactivity, feel free to reopen the issue if you still have the error
